plugins/woocommerce/client/legacy/css/woocommerce.scss

```diff
@@ -858,6 +862,7 @@ p.demo_store,
 				line-height: 1;
 				font-family: "WooCommerce";
 				content: "\e021";
+				content: "\e021" / "";
```

Comment:
`/ ""` is an alternative text for screen readers so they don't try to announce the "\e021" content.

**Reference**:
- https://www.w3.org/TR/css-content-3/#alt